src/org/ods/OdsPipeline.groovy

```diff
@@ -125,6 +128,7 @@ class OdsPipeline implements Serializable {
     context.displayNameUpdateEnabled = false
     context.ciSkipEnabled = false
     context.notifyNotGreen = false
+    context.sonarQubeBranch = '*'
```

Comment:
Doesn't this need a setter?

Reply:
@michaelsauter  - is in the below commits :)